packages/fast-form-generator-react/src/form/form.tsx

```diff
@@ -123,6 +124,10 @@ class Form extends React.Component<IFormProps & IManagedClasses<IFormClassNameCo
         }
     }
 
+    private isChildLocation(location: IFormLocation): boolean {
+        return location && location.dataLocation.slice(location.dataLocation.length - 8, location.dataLocation.length) === "children";
```

Comment:
Why does this subtract 8?

Reply:
because 8 is the length of the word "children", I'll add a comment